frontend/src/utils/polls/videos.ts

```diff
@@ -0,0 +1,55 @@
+import { TFunction } from 'react-i18next';
+import { EntitiesService, Entity, TypeEnum } from 'src/services/openapi';
+import { OrderedDialogs } from 'src/utils/types';
+
+let VIDEOS: Promise<Entity[]> | null = null;
+
+export function getTutorialVideos(): Promise<Entity[]> {
+  if (VIDEOS != null) {
+    return VIDEOS;
+  }
+  VIDEOS = EntitiesService.entitiesList({
+    type: TypeEnum.VIDEO,
+  }).then((data) => data.results ?? []);
```

Comment:
This api call returns as many videos as the configure PAGE_SIZE on the backend API. It may be slightly more performant to limit it to a smaller number around 6 ~ 10.

Reply:
Fixed by [ffe34d2](https://github.com/tournesol-app/tournesol/pull/885/commits/ffe34d2e0a4c10c1f7e09a49446c4ad428d1a785)